### PR TITLE
feat(sysk): integrated Nari Syskeeper 2000 as a new bridge backend 

### PR DESCRIPTION
**Pull request by lafirest**
Fixes [EMQX-10924](https://emqx.atlassian.net/browse/EMQX-10924)

### <samp>🤖 Generated by Copilot at 5c804a0</samp>

This pull request adds support for syskeeper forwarder and proxy, which are features that allow forwarding system metrics and events to a remote syskeeper server. It does so by adding a new application `emqx_bridge_syskeeper`, which contains the schema, client, frame, and proxy modules for the syskeeper protocol, and by updating the existing `emqx_bridge` and `emqx_connector` applications to include the new bridge and connector types. It also adds the license, documentation, and test files for the new application.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_bridge_syskeeper/include/emqx_bridge_syskeeper.hrl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
```
Comment (zmstone) [inline apps/emqx_bridge_syskeeper/include/emqx_bridge_syskeeper.hrl:2]:
```suggestion
%% Copyright (c) 2022-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
```

File: apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.app.src [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{application, emqx_bridge_syskeeper, [
+    {description, "EMQX Enterprise Bridge"},
```
Comment (zmstone) [inline apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.app.src:2]:
```suggestion
    {description, "EMQX Enterprise Data bridge for SysKeeper"},
```

File: apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl [hunk 3]
```diff
@@ -0,0 +1,133 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
```
Comment (zmstone) [inline apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl:2]:
```suggestion
%% Copyright (c) 2022-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
```

File: apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        {target_qos,
+            mk(
+                range(-1, 2),
+                #{desc => ?DESC("target_qos"), default => -1}
+            )},
+        {template,
+            mk(
+                binary(),
+                #{desc => ?DESC("template"), default => <<"${payload}">>}
+            )},
```
Comment (zmstone) [inline apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl:98]: we need to move these down to a struct of type named `parameters` 
wonder why this test did not fail for you:
https://github.com/emqx/emqx/blob/053c03f79f54d1667f9d045efca497ef61e92d92/apps/emqx_bridge/src/schema/emqx_bridge_v2_schema.erl#L166-L170

File: apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        target_qos => <<"-1">>,
+        template => <<"${payload}">>,
+        resource_opts => #{
+            worker_pool_size => 16
+        }
+    }.
+
+%% -------------------------------------------------------------------------------------------------
+%% Hocon Schema Definitions
+namespace() -> "bridge_syskeeper".
```
Comment (zmstone) [inline apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl:72]:
```suggestion
namespace() -> "syskeeper".
```

**Review by HJianBo — APPROVED** [review]

**Pull request merged**